ConcernsCaseWork/Service.TRAMS/Base/AbstractService.cs

```diff
@@ -15,19 +16,21 @@ namespace Service.TRAMS.Base
 	/// </summary>
 	public abstract class AbstractService
 	{
-		internal readonly IHttpClientFactory ClientFactory;
+		private readonly ICorrelationContext _correlationContext;
+		private readonly IHttpClientFactory _clientFactory;
 		private readonly ILogger<AbstractService> _logger;
-		internal const string HttpClientName = "TramsClient";
-		internal const string EndpointsVersion = "v2";
-		internal const string EndpointPrefix = "concerns-cases";
+		protected const string _httpClientName = "TramsClient";
```

Comment:
I've made some changes to this class in the decoupled version of concerns, so if it's agreed, would be putting these back to being properties:
https://github.com/DFE-Digital/amsd-casework/pull/407/files#diff-e5154a9be6c98acf69afd6dd560cad43c458a7369790bba6383192ead8fb86fa

Open to discussion, but thought it worth mentioning...

Reply:
Updated them to make things easier. 
But there's no base class to set the client name, so made it 
internal string HttpClientName { get; init; } = "TramsClient"; // was "Default";